accepted/0015-virtual-camera-support.md

```diff
@@ -0,0 +1,72 @@
+- Start Date: 2020-03-25
+- RFC PR: #15
+- Related Github Issue: https://github.com/obsproject/obs-studio/issues/25685
+
+# Summary
+
+Add the ability to render output to a virtual camera device on Linux, Mac, and Windows.
+
+# Motivation
+
+OBS is a powerful set of tools to manipulate live video streams that natively supports output to popular streaming services as well as rendering to a local video file. There are a huge number of people who engage in 1:1 or small-scale streaming using video conferencing software like Zoom or Google Hangouts. Many of these people have similar needs to those of traditional streamers, but for one reason or another cannot switch the video conferencing software they use (social graph, corporate policy). One way to deliver OBS output to these applications is to provide a virtual camera in OBS.
+
+Similar functionality: 
+* OBS can be extended by the [OBS-VirtualCam plugin](https://obsproject.com/forum/resources/obs-virtualcam.539/). Currently, only Windows is supported.
+* [Wirecast includes virtual camera support](http://www.telestream.net/pdfs/user-guides/Wirecast-8-User-Guide-Windows.pdf) on both Windows and Mac.
+* [It was possible](https://github.com/zakk4223/SyphonInject) to inject Syphon functionality into a process that draws to a GL context until some Mac security changes broke this.
+* [The Snap Camera](https://snapcamera.snapchat.com) is an example of an application that consumes input from a hardware webcam, processes the video stream, and outputs it in real time as a virtual device that appears in apps like Skype, Hangouts, or Zoom. It supports both Mac & PC.
+
+# Detailed design
+
+- Platform specific plugins shall be created that will be added as part of the main obs build.
+- Each plugin will start a server that will be responsible of transmitting the frame data to a client process.
+- A virtualcam "Driver" (client process) will be developed for each platform to consume data incomming from this plugins.
+- Each virtual driver shall be part of the installer with a checkbox to install the virtualcam driver
+
+## Cross-platform Interfaces
```

Comment:
I'm not really sure what this section is trying to say. I'm not convinced we need to have a generic interface for this. Can we not just have three different plugins that are wired up to the same "Start Virtual Cam" button depending on platform?